tests/FakeItEasy.Analyzer.Tests/UnusedReturnValueAnalyzerTests.cs

```diff
@@ -110,6 +110,39 @@ interface IFoo { int Bar(); }
         [Test]
         [SetUICulture("en-US")] // so that the message is in the expected language regardless of the OS language
         [SuppressMessage("Microsoft.Naming", "CA2204:Literals should be spelled correctly", MessageId = "CallTo", Justification = "It's an identifier")]
+        public void Diagnostic_Should_Be_Triggered_When_Call_Specification_Is_Not_Used_With_UsingStatic()

```

Comment:
Actually, the "Not" should be there, but it applies to the usage of the call specification... I copied and modified `Diagnostic_Should_Be_Triggered_When_Call_Specification_Is_Not_Used` and added `_With_UsingStatic` to the name, so the resulting name sounds confusing. I'll try to think of a better name.